addons/binding/org.openhab.binding.systeminfo.test/src/test/groovy/org/openhab/binding/systeminfo/test/SysteminfoOSGiTest.groovy

```diff
@@ -248,255 +288,333 @@ class SysteminfoOSGiTest extends OSGiTest{
         String acceptedItemType = "String";
 
         initializeThingWithChannel(channnelID,acceptedItemType);
-        assertItemState(acceptedItemType,DEFAULT_TEST_ITEM_NAME,DEFAULT_CHANNEL_TEST_PRIORITY);
+        assertItemState(acceptedItemType,DEFAULT_TEST_ITEM_NAME,DEFAULT_CHANNEL_TEST_PRIORITY, UnDefType.UNDEF);
     }
 
     @Test
     public void 'assert channel cpu#load is updated' () {
         String channnelID = SysteminfoBindingConstants.CHANNEL_CPU_LOAD
         String acceptedItemType = "Number";
 
+        DecimalType mockCpuLoadValue = new DecimalType(10.5)
```

Comment:
I think that the name mockedCpuLoadValue would be more proper, what do you think?